app/config/services/admin.xml

```diff
@@ -366,5 +366,19 @@
             <argument />
         </service>
 
+        <!--
+        ######################################################
+        # CV
+        ######################################################
+         -->
+
+        <service id="app.admin.summary_mission_type" class="AppBundle\Admin\MissionTypeAdmin">
```

Comment:
Do we really need a specific id here? it would be best to use the class only when possible.